services/app/src/components/avatarUploadModal/AvatarModalForm.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useRef, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Form } from 'antd';
+import Button from '../button/Button';
+import { useUserContext } from '../../contexts/UserContext';
+
+const AvatarModalForm = ({ moveToFinalStep }) => {
+  const [form] = Form.useForm();
+  const urlValue = Form.useWatch('url', form);
+
+  const [isPrimaryBtnDisabled, setIsPrimaryBtnDisabled] = useState(true);
+
+  const userContext = useUserContext();
+
+  const urlInputRef = useRef(null);
+
+  const handleKeyDown = (e) => {
+    if (e.code === 'Enter') {
+      moveToFinalStep(urlValue);
+    }
+  }
+
+  useEffect(() => {
+    userContext.setAvatarError(false);
+
+    if(urlInputRef.current) {
+      urlInputRef.current.focus();
```

Comment:
have you tried `autoFocus` property on input?

Reply:
Yes I have, for some reason it does not work. I think its because the input is wrapped in antD's form.

Reply:
Upadte: I saw, that it works in login page, but does not work in this modal, so I did a little research and cakme to an answer that autofocus does not work inside a modal. HTML `autoFocus` means focus input when the page loads. The solution is useRef. 
Source: https://github.com/ant-design/ant-design/issues/8668